src/pages/home.js

```diff
@@ -0,0 +1,119 @@
+            CloudWatch Agent is installed on the host. A configuration file is
+            generated for the agent to use with a specified number of logs
+            monitored. When the test begins, the agent is started and lines are
+            written to each log file monitored in the config. The lines are
+            written at a specified rate for the given test. While the agent is
+            running, it is also monitoring it's own resource usage so it can
+            report this data. These metrics are pulled from CloudWatch and saved
+            as the benchmarking data used on this website.
+          </p>
+        </section>
+        <section>
+          <h3>Need Assistance?</h3>
+          <p>
+            Read our FAQ below
+            <div>
+              <br></br>
+            </div>
+            Need further assistance? Let us know about your issue by opening an
+            issue on github:{" "}
+            <a
+              href="https://github.com/aws/amazon-cloudwatch-agent/issues/new/choose"
+              target="_blank"
+              rel="noreferrer"
+            >
+              here
+            </a>
+            <div>
+              <br></br>
+            </div>
+            Please include "CloudWatch Agent Performance Tracker" in the title
+            of the issue.
+          </p>
+        </section>
+        <section>
+          <h3>Intended Use</h3>
+          <p>
+            With this data, CloudWatch agent customers can get a better idea of
+            how the CloudWatch Agent performs under different loads. Since
+            several tests are run with different load cases, customers can
+            select a test that best represents a use case similar to theirs, and
+            view the benchmarked resources used by CloudWatch Agent with that
+            simulated configuration and load. This data should provide insights
+            into how efficiently the CloudWatch Agent can be expected to run
+            under a given load.
+            <div>
+              <br></br>
+            </div>
+            Currently, the performance benchmarking collects the CloudWatch
+            Agent's CPU usage and{" "}
+            <a
+              href="https://en.wikipedia.org/wiki/Resident_set_size"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Resident Set Size (RSS) memory
+            </a>
+            .
+          </p>
+        </section>
+        <section style={{ "margin-bottom": "3%" }}>
+          <h3> FAQ</h3>
+          <p>
+            Q: Graphs or Table are not updated when I refresh?
+            <br />
+            A: Hit this button{" "}
+            <button
+              title="Click"
+              style={{ width: "15%", height: "24px" }}
+              onClick={() => {
+                this.state.Receiver.cacheClear();
+              }}
+            >
+              Clear Cache
+            </button>
```

Comment:
This is a pretty blatant piece of tech debt. What is the issue that requires manual intervention to clear a cache for the user?

Reply:
So far we don't have any documented issue; however, in the worst case where something breaks or user wants fully restart the website ( maybe there is a git commit revert) this buttons provides that functionality. This is mostly a safety measure.